services/dkp-insights-management/0.1.2/helmrelease.yaml

```
@@ -0,0 +1,21 @@
+apiVersion: kustomize.toolkit.fluxcd.io/v1beta2
+kind: Kustomization
+metadata:
+  name: dkp-insights-management-helmrelease
+  namespace: ${releaseNamespace}
+spec:
+  force: false
+  prune: true
+  wait: true
+  interval: 6h
```

```suggestion
  interval: 6h
  retryInterval: 1m
```
actually we need this one